src/wrappers/PhylorefWrapper.js

```diff
@@ -784,24 +650,54 @@ class PhylorefWrapper {
       // in the form:
       //  phyloref:includes_TU some [internal1] and
       //  phyloref:includes_TU some [internal2] and ...
-      // To which we can then add the external specifiers.
-      if (internalSpecifiers.length === 1) {
-        logicalExpressions = this.createClassExpressionsForExternals(
-          phylorefAsJSONLD,
-          this.getIncludesRestrictionForTU(internalSpecifiers[0]),
-          externalSpecifiers,
-          []
+      //  phyloref:excludes_TU some [exclusion1] and
+      //  has_Ancestor some (phyloref:excludesTU some [exclusion2]) ...
+      //
+      // Since we don't know which of the external specifiers will actually
+      // be the one that will be used, we need to generate logical expressions
+      // for every possibility.
+
+      logicalExpressions = externalSpecifiers.map((selectedExternal) => {
+        // Add the internal specifiers.
+        const intersectionExprs = internalSpecifiers.map(
+          sp => this.getIncludesRestrictionForTU(sp)
         );
-      } else {
-        const expressionForInternals = {
+
+        // Add the selected external specifier.
+        intersectionExprs.push({
+          '@type': 'owl:Restriction',
+          onProperty: 'phyloref:excludes_TU',
+          someValuesFrom: new TaxonomicUnitWrapper(
+            selectedExternal,
+            this.defaultNomenCode
+          ).asOWLEquivClass,
+        });
+
+        // Collect all of the externals that are not selected.
+        const remainingExternals = externalSpecifiers.filter(ex => ex !== selectedExternal);
+
+        // Add the remaining externals, which we assume will resolve outside of
+        // this clade.
+        remainingExternals.forEach((externalTU) => {
+          intersectionExprs.push({
+            '@type': 'owl:Restriction',
+            onProperty: 'obo:CDAO_0000144', // has_Ancestor
```

Comment:
Note that there's a property `has_outside_TU` for this in the next release of the Phyloref Ontology: https://github.com/phyloref/phyloref-ontology/blob/master/phyloref.ofn#L38

Not sure whether using it should be a downstream change.

Reply:
Ah, I forgot about that property! I've opened an issue to make the change in Phyx.js once the new version of the Phyloref Ontology has been released: https://github.com/phyloref/phyx.js/issues/83.